### DISCUSSION TITLE
material list to plaintext

### DISCUSSION THREAD

**Discussion by pizzav-xyz** [maruohon/litematica#835] [General]
is there something like a website or mod to export materials of a schematic to plaintext

**Comment by sakura-ryoko** [comment]
You can export the Materials List right in Litematica to a text file.

**Reply by pizzav-xyz** [reply]
could you give instructions?

**Reply by sakura-ryoko** [reply]
When you are in the Material List screen, there is a button called "Write to File" at the top.

**Reply by sakura-ryoko** [reply]
From there, you can export it as a CSV for example, and convert it to whatever list format you want.

**Reply by pizzav-xyz** [reply]
thanks how do i close a disscusion

**Reply by sakura-ryoko** [reply]
I believe a maintainer needs to do it, unless it gives you a button to do it yourself.

**Reply by maruohon** [reply]
I think there should just be some kind of button to mark the question as answered by a given reply?

**Reply by pizzav-xyz** [reply]
i dont see that

**Reply by pizzav-xyz** [reply]
also is there an option to copy the path afterwards?

**Reply by maruohon** [reply]
I guess the category here matters? The discussions in `Q & A` have an `unanswered` label, unless they have an answer selected.

I think it prints the file name in chat? You can probably click that link to open the file? I don't remember for sure if it works in this context.
